applications/io_board/targets/freertos.armv6m.st-stm32g0b1rc-railcom/config.hxx

```diff
@@ -0,0 +1,79 @@
+#ifndef _APPLICATIONS_IO_BOARD_TARGET_CONFIG_HXX_
```

Comment:
There is no block header at the top of this file for copyright, file doxygen, etc.